octobot_commons/signals/signal_builder_wrapper.py

```diff
@@ -28,9 +28,11 @@ def __init__(
         identifier: str,
         signal_builder_class=signal_bundle_builder.SignalBundleBuilder,
         timeout: float = NO_TIMEOUT_VALUE,
+        builder_args: tuple = None
```

Comment:
Is there a reason to use a "custom" kwargs instead of using kwargs?

Reply:
you think we should use kwargs instead of args here ?

Reply:
I don't know, It seemed to me to correspond to the kwargs but it may not be the same thing

Reply:
these are regular python arguments, not keyword ones (meaning they are mandatory to some constructor) I did this is to avoid having to check that kwargs are filled in builder subclasses

Reply:
OK thanks. It seems logical to me now 👍